### PR TITLE
chore: remove `NetworkType` control flow

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Remove all instances of control flow based on `NetworkType`.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

Contributes to https://github.com/shapeshift/web/issues/1570

## Risk

Minimal, refactor-only.

## Testing

Ensure no chain-related regressions.

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Review by cjthompson — APPROVED** [review]

File: src/components/Modals/Send/Send.tsx [hunk 1]
```diff
@@ -36,11 +37,12 @@ export const SendModal = ({ asset, accountId }: SendModalProps) => {
             <Route
               path='/'
               component={(props: RouteComponentProps) => {
-                // TODO(gomes): make this cleaner
-                if (asset.chain === ChainTypes.Cosmos || asset.chain === ChainTypes.Osmosis) {
-                  return <CosmosForm asset={asset} accountId={accountId} {...props} />
-                }
-                return <Form asset={asset} accountId={accountId} {...props} />
+                const { chainNamespace } = fromChainId(asset.chainId)
+                return chainNamespace === CHAIN_NAMESPACE.Cosmos ? (
```
Comment (0xdef1cafe) [inline src/components/Modals/Send/Send.tsx:41]: missing osmosis?

**Review by 0xdef1cafe — DISMISSED** [review]

File: src/components/Modals/Send/Send.tsx (hunk 1 above)
Comment (0xApotheosis) [reply src/components/Modals/Send/Send.tsx:41]: They share the same `ChainNamespace`,  `cosmos`.

**Comment by 0xApotheosis** [comment]
https://github.com/shapeshift/web/blob/14ae670aac3539a2954dd5db9fc75271cd19971d/src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx#L232
> 
> You can destructure `chainId` from asset here and have a similar `chainId` discrimination and switch from hardcoded chainIds to the caip-exported ones, as you've done in this diff

Right you are, gomie - good spot.

Addressed in https://github.com/shapeshift/web/pull/1949/commits/3d2c6ae7e751d79421768fddb4e0eb68ec678890.

**Review by gomesalexandre — APPROVED** [review]
LGTM! Tested again with the previous testing criteria, and also tested sends, found no regression.
Well done 🍬 that new IIAFE is way cleaner than what we had before!

**Comment by 0xApotheosis** [comment]
> that new IIAFE is way cleaner than what we had before!

@gomesalexandre Immediately invoked awesome function expression?

**Review by cjthompson — APPROVED** [review]

**Pull request merged**